packages/terra-clinical-detail-view/tests/jest/DetailList.test.jsx

```diff
@@ -47,6 +47,21 @@ it('should set the heading level for the title', () => {
   expect(wrapper.html('.title')).toContain('<h2 class="title">Title</h2>');
 });
 
+it('should render a description list when isLabelValuePairList is true', () => {
+  const detailList = (
+    <DetailView.DetailList isLabelValuePairList>
+      <DetailView.DetailListItem item={(
+        <>
+          <dt>label</dt>
+          <dd>value</dd>
+        </>
+   )} />
+    </DetailView.DetailList>
+  );
+  const wrapper = render(detailList);
+  expect(wrapper).toMatchSnapshot();
```

Comment:
Can we also explicitly check for the `dl` element?

Reply:
Updated here: https://github.com/cerner/terra-clinical/pull/864/commits/aa3c94aa8e25e9e20104ac1e5d07d744d4901b82